### PR TITLE
Adding diffpy.fourigui

### PR DESCRIPTION
**Pull request by bobleesj**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by bobleesj** [comment]
@sbillinge Could you please write a comment "confirming thy are willing to be listed"? 

We will request for review once we have a full PyPI release next week with docs/readme double checked.

Now, I am using the test pypi version to ensure we have no hiccups next week.

**Comment by bobleesj** [comment]
@conda-forge/help-python

For linux only, we are hoping to install `sudo apt-get install -y xvfb` for running full `pytest`. What would be the best to install it while maintaining the `noarch: python` status? I tried to create `build.sh` file containing 

```
#!/bin/bash
sudo apt-get update
sudo apt-get install -y xvfb
python -m pip install --no-deps --ignore-installed .
```

but I had no luck. Thanks!

**Comment by sbillinge** [comment]
I am willing to be maintainer on this package

File: recipes/diffpy.fourigui/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+test: 
+  requires:
+    - pip
+    - pytest
+  source_files:
+    - tests
+  commands:
+    - conda list
+    - pip list
```
Comment (ocefpaf) [inline recipes/diffpy.fourigui/meta.yaml:37]: I believe you don't need this. The list of packages is in the log and this is not a test of the package.

```suggestion
```

File: recipes/diffpy.fourigui/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{% set name = "diffpy.fourigui" %}
+{% set version = "0.1.0rc2" %}
```
Comment (ocefpaf) [inline recipes/diffpy.fourigui/meta.yaml:2]: This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

**Comment by ocefpaf** [comment]
> For linux only, we are hoping to install `sudo apt-get install -y xvfb` for running full `pytest`. What would be the best to install it while maintaining the `noarch: python` status? I tried to create `build.sh` file containing

You can't sudo install anything in a build. that could contaminate the packages with unknown dependencies and lead to a broken package. Please see https://conda-forge.org/docs/maintainer/knowledge_base/#yum_requirementstxt for this and search the existing dependencies for `xvfb` examples.

File: recipes/diffpy.fourigui/meta.yaml (hunk 2 above)
Comment (bobleesj) [reply recipes/diffpy.fourigui/meta.yaml:2]: Fixed - `conda_build_config.yaml` created for `rc` release.

File: recipes/diffpy.fourigui/meta.yaml (hunk 1 above)
Comment (bobleesj) [reply recipes/diffpy.fourigui/meta.yaml]: Yes, removed.

**Comment by bobleesj** [comment]
@conda-forge/help-python @ocefpaf Ready for review! Thanks for the `yum_requirements.txt` suggestion above. Works as intended.

**Pull request merged**